examples/layers/preprocessing/cut_out_demo.py

```diff
@@ -0,0 +1,47 @@
+"""cut_out_demo.py shows how to use the CutOut preprocessing layer.
```

Comment:
Please save some sample images generated by this script and link them in the PR.